lib/site_prism/element_checker.rb

```diff
@@ -1,9 +1,35 @@
 module SitePrism
   module ElementChecker
+
     def all_there?
       Capybara.using_wait_time(0) do
-        self.class.mapped_items.all? { |element| send "has_#{element}?" }
+        expected_elements.all? { |element| send "has_#{element}?" }
       end
     end
+
+    def excluded_all_absent?
```

Comment:
I did not end up using this method a whole lot, but I thought there might be at least some cases where one would want to test whether or not the elements in excluded_elements are actually absent.